### PR TITLE
updated release notes

### PR DESCRIPTION
**Pull request by jasonstephen15**

### CODE REVIEW

File: microsoft-edge/webview2/releasenotes.md [hunk 1]
```diff
@@ -40,6 +43,12 @@ Release Date: December 8, 2020
...
 *   Updated `NewWindowRequested` event to handle `Ctrl`+`click` and `Shift`+`click`.  \([\#168][GithubMicrosoftedgeWebviewfeedbackIssue168] and [\#371][GithubMicrosoftedgeWebviewfeedbackIssue371]\).  
+*   Promoted the following experimental changes to stable:
+    * [WebResourceResponseReceived API](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2_2?view=webview2-1.0.721-prerelease#add_webresourceresponsereceived)
+    * [NavigateWithWebResourceRequest API](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2environment2?view=webview2-1.0.721-prerelease#createwebresourcerequest)
+    * [Cookie management API](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2cookiemanager?view=webview2-1.0.721-prerelease)
+    * [DOMContentLoaded API](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2_2?view=webview2-1.0.721-prerelease#add_domcontentloaded)
+    * Exposed [environment property](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2_2?view=webview2-1.0.721-prerelease#get_environment) from ICoreWebView2
```
Comment (Reezaali) [inline microsoft-edge/webview2/releasenotes.md:51]: Can we change "exposed" to "made public"?

**Review by Reezaali — CHANGES_REQUESTED** [review]
Minor change suggested.

**Comment by jasonstephen15** [comment]
ready to merge!

**Review by jasonstephen15 — COMMENTED** [review]
changes in

**Comment by jasonstephen15** [comment]
ready to merge

**Review by Reezaali — APPROVED** [review]

**Pull request merged**